static/js/components/LearnerSearch.js

```diff
@@ -220,7 +226,7 @@ export default class LearnerSearch extends SearchkitComponent {
             title="Final Grade in Selected Course"
```

Comment:
Should we also remove the final grade and course title filters for non-staff users?